superset/models/alerts.py

```diff
@@ -44,23 +50,29 @@
 )
 
 
+class AlertValidatorType(str, enum.Enum):
+    not_null = "Not Null"
+    gte_threshold = ">="
```

Comment:
unify gte_threshold, lte_threshold into the operator
e.g. 
operator(value, threshold)
operator can be <. <=, ==, >, >=